dell/xps/15-9530/fingerprint/default.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs, lib, config, ... }: {
+
+  environment.systemPackages = [
+
+  ];
+
+  services.fprintd = {
+    enable = true;
+    tod = {
+      enable = true;
+      driver = pkgs.libfprint-2-tod1-goodix;
+    };
```

Comment:
Is this section required on the 9530 ? The service works find on 9520 without specifying the `tod`.